extensions/notebook/src/dialog/managePackages/installedPackagesTab.ts

```diff
@@ -32,18 +33,27 @@ export class InstalledPackagesTab {
 		this.installedPkgTab = azdata.window.createTab(localize('managePackages.installedTabTitle', "Installed"));
 
 		this.installedPkgTab.registerContent(async view => {
-			let dropdownValues: string[];
-			if (this.dialog.currentPkgType === PythonPkgType.Anaconda) {
-				dropdownValues = [PythonPkgType.Anaconda, PythonPkgType.Pip];
-			} else {
-				dropdownValues = [PythonPkgType.Pip];
-			}
+
+			// TODO: only supporting single location for now. We should add a drop down for multi locations mode
+			//
```

Comment:
Better to just create an actual issue in github

Reply:
I was not sure  if we want to support that at all or not in future, I designed the model to support multiple but didn't want to make decision for UI right now. I'll add a task for it

---
In reply to: [350874329](https://github.com/microsoft/azuredatastudio/pull/8473#discussion_r350874329) [](ancestors = 350874329)